src/components/item/item.md.scss

```diff
@@ -64,7 +63,8 @@ $item-md-sliding-content-background: $list-md-background-color !default;
   box-shadow: none;
   transition: background-color 300ms cubic-bezier(.4, 0, .2, 1);
 }
```

Comment:
Oh no mistake, just odd code (normally, try to keep a single selector with all of that selector's properties).
In this case, you have on line 54: `.item-md` and same on line 67.
Removing lines 65-67 and perhaps adding a single breakline between `transition` and the `padding` will simplify the code, and make it better to read.